pkg/acl/acl.go

```diff
@@ -265,37 +171,37 @@ func ApplyAccessControlPolicies(ctx context.Context, operation string, httpVerb
 		return false, errMessage
 	}
 
-	action, actionPolicy := IsOperationAllowedByAccessControlPolicy(spiffeID, appID, operation, httpVerb, isHTTP, acl)
-	emitACLMetrics(actionPolicy, appID, trustDomain, namespace, operation, httpVerb.String(), action)
+	action, actionPolicy := isOperationAllowedByAccessControlPolicy(spiffeID, operation, httpVerb, isHTTP, acl)
+	emitACLMetrics(spiffeID, actionPolicy, operation, httpVerb.String(), action)
 
 	if !action {
-		errMessage = fmt.Sprintf("access control policy has denied access to appid: %s operation: %s verb: %s", appID, operation, httpVerb)
-		log.Debugf(errMessage)
+		errMessage = fmt.Sprintf("access control policy has denied access to id: %s operation: %s verb: %s", spiffeID.URL(), operation, httpVerb)
+		log.Debug(errMessage)
```

Comment:
Why L170 use `log.Debugf` but use `log.Debug` here?

Reply:
`log.Debugf` is like fmt.Sprintf. In this case we are passing the exact message so we must use `log.Debug`.